containers/ecr-viewer/src/app/api/tests/listEcrDataService.test.tsx

```diff
@@ -154,166 +90,117 @@ describe("listEcrDataService", () => {
-        patient_report_date: "06/21/2024 8:00 AM EDT",
-        reportable_condition: "stuff",
-        rule_summary: "yup",
-      },
-    ]);
-  });
-
-  it("should get data from the fhir_metadata table", async () => {
-    process.env.SOURCE = "postgres";
-    process.env.NEXT_PUBLIC_NON_INTEGRATED_VIEWER = "true";
-    database.manyOrNone<{
-      ecr_id: string;
-      date_created: string;
-      patient_birth_date: string;
-      patient_name_first: string;
-      patient_name_last: string;
-      report_date: string;
-      reportable_condition: string;
-      rule_summary: string;
-    }> = jest.fn(() =>
-      Promise.resolve<CompleteEcrDataModel[]>([
-        {
-          ecr_id: "1234",
-          date_created: "2024-06-21T12:00:00Z",
-          patient_name_first: "boy",
-          patient_name_last: "lnam",
-          patient_birth_date: new Date("1990-01-01T05:00:00.000Z"),
-          report_date: new Date("2024-06-20T04:00:00.000Z"),
-          reportable_condition: "sick",
-          rule_summary: "stuff",
-          data: "",
-          data_link: "",
-          data_source: "DB",
-        },
-      ]),
-    );
-
-    const actual: EcrDisplay[] = await listEcrData();
+  describe("list Ecr data", () => {
+    it("should return empty array when no data is found", async () => {
+      let startIndex = 0;
+      let itemsPerPage = 25;
+      database.manyOrNone = jest.fn(() => Promise.resolve([]));
+      const actual = await listEcrData(startIndex, itemsPerPage);
+      expect(database.manyOrNone).toHaveBeenCalledExactlyOnceWith(
+        "SELECT ecr_id, patient_name_first, patient_name_last, patient_birth_date, report_date, reportable_condition, rule_summary, date_created FROM fhir_metadata order by date_created DESC OFFSET 0 ROWS FETCH NEXT 25 ROWS ONLY",
+      );
+      expect(actual).toBeEmpty();
+    });
 
-    expect(database.manyOrNone).toHaveBeenCalledExactlyOnceWith(
-      "SELECT fhir.ecr_id, date_created, patient_name_first, patient_name_last, patient_birth_date, report_date, reportable_condition, rule_summary FROM fhir LEFT OUTER JOIN fhir_metadata on fhir.ecr_id = fhir_metadata.ecr_id order by date_created DESC",
-    );
-    expect(actual).toEqual([
-      {
-        date_created: "06/21/2024 8:00 AM EDT",
-        ecrId: "1234",
-        patient_date_of_birth: "01/01/1990",
-        patient_first_name: "boy",
-        patient_last_name: "lnam",
-        patient_report_date: "06/20/2024 12:00 AM EDT",
-        reportable_condition: "sick",
-        rule_summary: "stuff",
-      },
-    ]);
-  });
+    it("should return data when found", async () => {
+      database.manyOrNone<{
+        ecr_id: string;
+        date_created: string;
+        patient_birth_date: string;
+        patient_name_first: string;
+        patient_name_last: string;
+        report_date: string;
+        reportable_condition: string;
+        rule_summary: string;
+      }> = jest.fn(() =>
+        Promise.resolve<EcrMetadataModel[]>([
```

Comment:
Could this type here be replaced with just `EcrMetadataModel`?
```suggestion
      database.manyOrNone<EcrMetadataModel> = jest.fn(() =>
        Promise.resolve<EcrMetadataModel[]>([
```